src/core/Akka.Tests/Actor/ActorSelectionSpec.cs

```diff
@@ -39,6 +39,15 @@ public void Can_resolve_child_path()
         }
 
         [Fact]
+        public void Can_resolve_absolute_path()
```

Comment:
Test case for local actor selections using full actor path, complete with akka://MySystem etc...